repos/fdbt-site/src/design/modules/_layout.scss

```diff
@@ -332,6 +332,7 @@ svg:not(:root) {
 
     .govuk-form-group {
         margin-bottom: 0;
+        max-width: 230px;
```

Comment:
This seems like a change that could affect a large number of pages across the site?

Reply:
I think because it's a subclass within `.time-restrictions-table` it should only affect form groups that are children of elements with this class? And `.time-restrictions-table` only lives on this page and the original time restrictions page. Does that make sense?

Reply:
Ah cool makes sense 👍